fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/item/PlayerInventoryStorage.java

```diff
@@ -43,6 +43,7 @@
 @ApiStatus.Experimental
 @Deprecated
 @ApiStatus.NonExtendable
+// TODO: should sync stacks by sending a packet, like PlayerInventory#offer.
```

Comment:
Note to self: this needs to be addressed.

Reply:
I've updated the TODO. Vanilla doesn't do it reliably, so I'll wait for an issue report before changing it if that's necessary.